api-admin/src/main/resources/api/openapi.yml

```diff
@@ -129,14 +129,13 @@ paths:
           content: { application/json: { schema: {type: object} } }
         '500':
           $ref: '#/components/responses/ServerError'
-  /api/portals/v1/{portalShortcode}/surveys/{stableId}:
+  /api/portals/v1/{portalShortcode}/surveys:
     post:
```

Comment:
I think this is appropriate. The other option to consider would be keeping `{stableId}` and using `PUT`, which would be nicely RESTful.